lib/ohai/plugins/solaris2/network.rb

```diff
@@ -101,7 +101,8 @@ def full_interface_name(iface, part_name, index)
     cint = nil
 
     so.stdout.lines do |line|
-      if line =~ /^([0-9a-zA-Z\.\:\-]+)\S/
+      # if line =~ /^([0-9a-zA-Z\.\:\-]+)\S/
+      if line =~ /^([0-9a-zA-Z\.\:\-]+): \S+ mtu (\d+) index (\d+)/
```

Comment:
Thanks.  Will include the link in the update.   
While looking at the history of this change it seems like the +mtu and +index was unintentionally removed while fixing the regex.